### ISSUE TITLE
west build -p auto -b nrf9160_pca10015ns fails with undeclared symbols

### ISSUE THREAD

**Issue by coderbyheart** [bifravst/cat-tracker-fw#15]
This seems to be the issue:

```
[43/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/spm.c.obj
FAILED: spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/spm.c.obj 
/root/ncs/gcc-arm-none-eabi-7-2018-q2-update/bin/arm-none-eabi-gcc -DBUILD_VERSION=v1.14.99-ncs3-snapshot2-1265-g736c7dd7525b -DKERNEL -DMBEDTLS_CONFIG_FILE=\"nrf-config.h\" -DNRF9160_XXAA -DUSE_PARTITION_MANAGER=1 -D_FORTIFY_SOURCE=2 -D__ZEPHYR__=1 -I/root/ncs/zephyr/kernel/include -I/root/ncs/zephyr/arch/arm/include -I/root/ncs/zephyr/include -I/root/ncs/zephyr/include/drivers -Ispm/zephyr/include/generated -I/root/ncs/zephyr/soc/arm/nordic_nrf/nrf91 -I/root/ncs/zephyr/soc/arm/nordic_nrf/include -I/root/ncs/zephyr/lib/libc/minimal/include -I/root/ncs/zephyr/ext/hal/cmsis/Include -I/root/ncs/zephyr/ext/hal/nordic/nrfx -I/root/ncs/zephyr/ext/hal/nordic/nrfx/drivers/include -I/root/ncs/zephyr/ext/hal/nordic/nrfx/hal -I/root/ncs/zephyr/ext/hal/nordic/nrfx/mdk -I/root/ncs/zephyr/ext/hal/nordic/. -I/root/ncs/nrf/include -I/root/ncs/nrfxlib/crypto/nrf_cc310_mbedcrypto/include/mbedtls -Ispm/zephyr/modules/nrfxlib/nrf_security/include -I/root/ncs/mbedtls/include -I/root/ncs/mbedtls/include/mbedtls -isystem /root/ncs/gcc-arm-none-eabi-7-2018-q2-update/bin/../lib/gcc/arm-none-eabi/7.3.1/include -isystem /root/ncs/gcc-arm-none-eabi-7-2018-q2-update/bin/../lib/gcc/arm-none-eabi/7.3.1/include-fixed -Os -nostdinc -imacros/root/ncs/nrf/applications/cat_tracker/build/spm/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -mthumb -mcpu=cortex-m33 -mfpu=fpv5-sp-d16 -mfloat-abi=hard -imacros/root/ncs/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -fno-reorder-functions -fno-defer-pop -ffunction-sections -fdata-sections -mabi=aapcs -march=armv8-m.main+dsp -mcmse -std=c99 -MD -MT spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/spm.c.obj -MF spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/spm.c.obj.d -o spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/spm.c.obj   -c /root/ncs/nrf/subsys/spm/spm.c
/root/ncs/nrf/subsys/spm/spm.c: In function 'spm_config_nsc_flash':
/root/ncs/nrf/subsys/spm/spm.c:129:2: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
  NRF_SPU->FLASHNSC[0].REGION = FLASH_NSC_REGION_FROM_ADDR(__sg_start);
  ^~~~~~~
  NRF_SPU_S
/root/ncs/nrf/subsys/spm/spm.c:129:2: note: each undeclared identifier is reported only once for each function it appears in
/root/ncs/nrf/subsys/spm/spm.c: In function 'spm_config_flash':
/root/ncs/nrf/subsys/spm/spm.c:160:3: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
   NRF_SPU->FLASHREGION[i].PERM = flash_perm[i];
   ^~~~~~~
   NRF_SPU_S
/root/ncs/nrf/subsys/spm/spm.c: In function 'spm_config_sram':
/root/ncs/nrf/subsys/spm/spm.c:207:3: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
   NRF_SPU->RAMREGION[i].PERM = sram_perm[i];
   ^~~~~~~
   NRF_SPU_S
/root/ncs/nrf/subsys/spm/spm.c: In function 'usel_or_split':
/root/ncs/nrf/subsys/spm/spm.c:224:21: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
  const u32_t perm = NRF_SPU->PERIPHID[id].PERM;
                     ^~~~~~~
                     NRF_SPU_S
/root/ncs/nrf/subsys/spm/spm.c: In function 'spm_config_peripheral':
/root/ncs/nrf/subsys/spm/spm.c:262:3: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
   NRF_SPU->PERIPHID[id].PERM = PERIPH_PRESENT | PERIPH_NONSEC |
   ^~~~~~~
   NRF_SPU_S
/root/ncs/nrf/subsys/spm/spm.c: In function 'spm_config_peripherals':
/root/ncs/nrf/subsys/spm/spm.c:321:3: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
   NRF_SPU->GPIOPORT[0].PERM = 0;
   ^~~~~~~
   NRF_SPU_S
```

<details>
<summary>full log</summary>

```
-- west build: making build dir /root/ncs/nrf/applications/cat_tracker/build pristine
-- west build: build configuration:
       source directory: /root/ncs/nrf/applications/cat_tracker
       build directory: /root/ncs/nrf/applications/cat_tracker/build
       BOARD: nrf9160_pca10015ns (origin: command line)
-- west build: generating a build system
-- Using application from '/root/ncs/nrf/applications/cat_tracker'
Zephyr version: 1.14.99
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.5.3", minimum required is "3.4") 
-- Selected BOARD nrf9160_pca10015ns
-- Found west: /root/.local/bin/west (found suitable version "0.6.0", minimum required is "0.6.0")
-- Cache files will be written to: /root/.cache/zephyr
USING OUT OF TREE BOARD
-- Loading /root/ncs/nrf/boards/arm/nrf9160_pca10015/nrf9160_pca10015ns.dts as base
-- Overlaying /root/ncs/zephyr/dts/common/common.dts
Device tree configuration written to /root/ncs/nrf/applications/cat_tracker/build/zephyr/include/generated/generated_dts_board.conf
Parsing Kconfig tree in /root/ncs/nrf/applications/cat_tracker/Kconfig
Loaded configuration '/root/ncs/nrf/boards/arm/nrf9160_pca10015/nrf9160_pca10015ns_defconfig'
Merged configuration '/root/ncs/nrf/applications/cat_tracker/prj.conf'
Configuration saved to '/root/ncs/nrf/applications/cat_tracker/build/zephyr/.config'
-- The C compiler identification is GNU 7.3.1
-- The CXX compiler identification is GNU 7.3.1
-- The ASM compiler identification is GNU
-- Found assembler: /root/ncs/gcc-arm-none-eabi-7-2018-q2-update/bin/arm-none-eabi-gcc
-- Performing Test toolchain_is_ok
-- Performing Test toolchain_is_ok - Success
Including module: nrf in path: /root/ncs/nrf
-- Using application from '/root/ncs/nrf/samples/nrf9160/spm'
Zephyr version: 1.14.99
USING OUT OF TREE BOARD
-- Loading /root/ncs/nrf/boards/arm/nrf9160_pca10015/nrf9160_pca10015ns.dts as base
-- Overlaying /root/ncs/zephyr/dts/common/common.dts
Device tree configuration written to /root/ncs/nrf/applications/cat_tracker/build/spm/zephyr/include/generated/generated_dts_board.conf

warning: TRUSTED_EXECUTION_NONSECURE (defined at arch/Kconfig:146) was assigned the value 'y' but
got the value 'n'. You can check symbol information (including dependencies) in the 'menuconfig'
interface (see the Application Development Primer section of the manual), or in the Kconfig
reference at
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_TRUSTED_EXECUTION_NONSECURE.html
(which is updated regularly from the master branch). See the 'Setting configuration values' section
of the Board Porting Guide as well.

warning: the choice symbol BOARD_NRF9160_PCA10015NS (defined at
/root/ncs/nrf/boards/arm/nrf9160_pca10015/Kconfig.board:13) was selected (set =y), but
BOARD_NRF9160_PCA10015 (defined at /root/ncs/nrf/boards/arm/nrf9160_pca10015/Kconfig.board:9) ended
up as the choice selection.  You can check symbol information (including dependencies) in the
'menuconfig' interface (see the Application Development Primer section of the manual), or in the
Kconfig reference at
http://docs.zephyrproject.orParsing Kconfig tree in /root/ncs/zephyr/Kconfig
Loaded configuration '/root/ncs/nrf/boards/arm/nrf9160_pca10015/nrf9160_pca10015ns_defconfig'
Merged configuration '/root/ncs/nrf/samples/nrf9160/spm/prj.conf'
Configuration saved to '/root/ncs/nrf/applications/cat_tracker/build/spm/zephyr/.config'
g/latest/reference/kconfig/CONFIG_BOARD_NRF9160_PCA10015NS.html (which
is updated regularly from the master branch). See the 'Setting configuration values' section of the
Board Porting Guide as well.
Including module: nrf in path: /root/ncs/nrf
Including module: nffs in path: /root/ncs/modules/fs/nffs
Including module: segger in path: /root/ncs/modules/debug/segger
Including module: mbedtls in path: /root/ncs/modules/crypto/mbedtls
Including module: mcuboot in path: /root/ncs/mcuboot/zephyr
Including module: mcumgr in path: /root/ncs/modules/lib/mcumgr
Including module: tinycbor in path: /root/ncs/modules/lib/tinycbor
Including module: nrfxlib in path: /root/ncs/nrfxlib
Including module: nffs in path: /root/ncs/modules/fs/nffs
Including module: segger in path: /root/ncs/modules/debug/segger
Including module: mbedtls in path: /root/ncs/modules/crypto/mbedtls
Including module: mcuboot in path: /root/ncs/mcuboot/zephyr
Including module: mcumgr in path: /root/ncs/modules/lib/mcumgr
Including module: tinycbor in path: /root/ncs/modules/lib/tinycbor
Including module: nrfxlib in path: /root/ncs/nrfxlib
Running Partition Manager...
-- Configuring done
-- Generating done
-- Build files have been written to: /root/ncs/nrf/applications/cat_tracker/build
-- west build: building application
[1/359] Preparing syscall dependency handling

[2/359] Preparing syscall dependency handling

[3/359] Generating include/generated/syscall_macros.h
[4/359] Generating include/generated/syscall_macros.h
[5/359] Generating misc/generated/syscalls_subdirs.trigger
[6/359] Generating misc/generated/syscalls_subdirs.trigger
[7/359] Generating include/generated/driver-validation.h
[8/359] Generating include/generated/kobj-types-enum.h, include/generated/otype-to-str.h
[9/359] Generating include/generated/kobj-types-enum.h, include/generated/otype-to-str.h
[10/359] Generating include/generated/driver-validation.h
[11/359] Generating misc/generated/syscalls.json
[12/359] Generating misc/generated/syscalls.json
[13/359] Generating include/generated/syscall_dispatch.c, include/generated/syscall_list.h
[14/359] Generating include/generated/syscall_dispatch.c, include/generated/syscall_list.h
[15/359] Building C object spm/zephyr/CMakeFiles/spm_offsets.dir/arch/arm/core/offsets/offsets.c.obj
[16/359] Generating include/generated/offsets.h
[17/359] Generating linker.cmd
[18/359] Building C object spm/CMakeFiles/spm_app.dir/src/main.c.obj
[19/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/crc32_sw.c.obj
[20/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/crc16_sw.c.obj
[21/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/crc8_sw.c.obj
[22/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/timeutil.c.obj
[23/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/arch/common/isr_tables.c.obj
[24/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/crc7_sw.c.obj
[25/359] Building C object spm/zephyr/modules/nrfxlib/nrf_security/src/mbedtls/CMakeFiles/spm_mbedcrypto_vanilla.dir/root/ncs/mbedtls/library/platform.c.obj
[26/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/hex.c.obj
[27/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/arch/common/sw_isr_common.c.obj
[28/359] Linking C static library spm/spm_app/libspm_app.a
[29/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/misc/generated/configs.c.obj
[30/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/arch/arm/core/cortex_m/tz/arm_core_tz.c.obj
[31/359] Building C object spm/zephyr/modules/nrfxlib/nrf_security/src/mbedtls/CMakeFiles/spm_mbedcrypto_vanilla.dir/root/ncs/mbedtls/library/pkwrite.c.obj
[32/359] Building ASM object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/swap_helper.S.obj
[33/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr_prebuilt.dir/misc/empty_file.c.obj
[34/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/arch/arm/core/cortex_m/cmse/arm_core_cmse.c.obj
[35/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/thread_entry.c.obj
[36/359] Building ASM object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/cpu_idle.S.obj
[37/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/fdtable.c.obj
[38/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/ext/hal/nordic/nrfx/mdk/system_nrf9160.c.obj
[39/359] Building ASM object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/exc_exit.S.obj
[40/359] Building ASM object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/fault_s.S.obj
[41/359] Building ASM object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/nmi_on_reset.S.obj
[42/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/work_q.c.obj
[43/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/spm.c.obj
FAILED: spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/spm.c.obj 
/root/ncs/gcc-arm-none-eabi-7-2018-q2-update/bin/arm-none-eabi-gcc -DBUILD_VERSION=v1.14.99-ncs3-snapshot2-1265-g736c7dd7525b -DKERNEL -DMBEDTLS_CONFIG_FILE=\"nrf-config.h\" -DNRF9160_XXAA -DUSE_PARTITION_MANAGER=1 -D_FORTIFY_SOURCE=2 -D__ZEPHYR__=1 -I/root/ncs/zephyr/kernel/include -I/root/ncs/zephyr/arch/arm/include -I/root/ncs/zephyr/include -I/root/ncs/zephyr/include/drivers -Ispm/zephyr/include/generated -I/root/ncs/zephyr/soc/arm/nordic_nrf/nrf91 -I/root/ncs/zephyr/soc/arm/nordic_nrf/include -I/root/ncs/zephyr/lib/libc/minimal/include -I/root/ncs/zephyr/ext/hal/cmsis/Include -I/root/ncs/zephyr/ext/hal/nordic/nrfx -I/root/ncs/zephyr/ext/hal/nordic/nrfx/drivers/include -I/root/ncs/zephyr/ext/hal/nordic/nrfx/hal -I/root/ncs/zephyr/ext/hal/nordic/nrfx/mdk -I/root/ncs/zephyr/ext/hal/nordic/. -I/root/ncs/nrf/include -I/root/ncs/nrfxlib/crypto/nrf_cc310_mbedcrypto/include/mbedtls -Ispm/zephyr/modules/nrfxlib/nrf_security/include -I/root/ncs/mbedtls/include -I/root/ncs/mbedtls/include/mbedtls -isystem /root/ncs/gcc-arm-none-eabi-7-2018-q2-update/bin/../lib/gcc/arm-none-eabi/7.3.1/include -isystem /root/ncs/gcc-arm-none-eabi-7-2018-q2-update/bin/../lib/gcc/arm-none-eabi/7.3.1/include-fixed -Os -nostdinc -imacros/root/ncs/nrf/applications/cat_tracker/build/spm/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -mthumb -mcpu=cortex-m33 -mfpu=fpv5-sp-d16 -mfloat-abi=hard -imacros/root/ncs/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -fno-reorder-functions -fno-defer-pop -ffunction-sections -fdata-sections -mabi=aapcs -march=armv8-m.main+dsp -mcmse -std=c99 -MD -MT spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/spm.c.obj -MF spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/spm.c.obj.d -o spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/spm.c.obj   -c /root/ncs/nrf/subsys/spm/spm.c
/root/ncs/nrf/subsys/spm/spm.c: In function 'spm_config_nsc_flash':
/root/ncs/nrf/subsys/spm/spm.c:129:2: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
  NRF_SPU->FLASHNSC[0].REGION = FLASH_NSC_REGION_FROM_ADDR(__sg_start);
  ^~~~~~~
  NRF_SPU_S
/root/ncs/nrf/subsys/spm/spm.c:129:2: note: each undeclared identifier is reported only once for each function it appears in
/root/ncs/nrf/subsys/spm/spm.c: In function 'spm_config_flash':
/root/ncs/nrf/subsys/spm/spm.c:160:3: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
   NRF_SPU->FLASHREGION[i].PERM = flash_perm[i];
   ^~~~~~~
   NRF_SPU_S
/root/ncs/nrf/subsys/spm/spm.c: In function 'spm_config_sram':
/root/ncs/nrf/subsys/spm/spm.c:207:3: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
   NRF_SPU->RAMREGION[i].PERM = sram_perm[i];
   ^~~~~~~
   NRF_SPU_S
/root/ncs/nrf/subsys/spm/spm.c: In function 'usel_or_split':
/root/ncs/nrf/subsys/spm/spm.c:224:21: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
  const u32_t perm = NRF_SPU->PERIPHID[id].PERM;
                     ^~~~~~~
                     NRF_SPU_S
/root/ncs/nrf/subsys/spm/spm.c: In function 'spm_config_peripheral':
/root/ncs/nrf/subsys/spm/spm.c:262:3: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
   NRF_SPU->PERIPHID[id].PERM = PERIPH_PRESENT | PERIPH_NONSEC |
   ^~~~~~~
   NRF_SPU_S
/root/ncs/nrf/subsys/spm/spm.c: In function 'spm_config_peripherals':
/root/ncs/nrf/subsys/spm/spm.c:321:3: error: 'NRF_SPU' undeclared (first use in this function); did you mean 'NRF_SPU_S'?
   NRF_SPU->GPIOPORT[0].PERM = 0;
   ^~~~~~~
   NRF_SPU_S
[44/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/soc/arm/nordic_nrf/nrf91/mpu_regions.c.obj
[45/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/subsys/power/reboot.c.obj
[46/359] Building ASM object spm/zephyr/arch/arm/core/cortex_m/CMakeFiles/spm_arch__arm__core__cortex_m.dir/vector_table.S.obj
[47/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/sem.c.obj
[48/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/mempool.c.obj
[49/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/soc/arm/nordic_nrf/nrf91/soc.c.obj
[50/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/drivers/timer/sys_clock_init.c.obj
[51/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/subsys/spm/secure_services.c.obj
[52/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/rb.c.obj
[53/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/drivers/console/uart_console.c.obj
[54/359] Building ASM object spm/zephyr/arch/arm/core/cortex_m/CMakeFiles/spm_arch__arm__core__cortex_m.dir/reset.S.obj
[55/359] Building ASM object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/isr_wrapper.S.obj
[56/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/subsys/power/policy/policy_residency.c.obj
[57/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/subsys/power/power.c.obj
[58/359] Building C object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/swap.c.obj
[59/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/root/ncs/nrf/drivers/adp536x/adp536x.c.obj
[60/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/ext/hal/nordic/nrfx_glue.c.obj
[61/359] Building C object spm/zephyr/lib/libc/minimal/CMakeFiles/spm_lib__libc__minimal.dir/source/stdlib/atoi.c.obj
[62/359] Building C object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/irq_manage.c.obj
[63/359] Building C object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/thread.c.obj
[64/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/drivers/timer/nrf_rtc_timer.c.obj
[65/359] Building C object spm/zephyr/lib/libc/minimal/CMakeFiles/spm_lib__libc__minimal.dir/source/stdlib/strtoul.c.obj
[66/359] Building C object spm/zephyr/lib/libc/minimal/CMakeFiles/spm_lib__libc__minimal.dir/source/stdlib/strtol.c.obj
[67/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/drivers/clock_control/nrf_power_clock.c.obj
[68/359] Building C object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/prep_c.c.obj
[69/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/lib/os/printk.c.obj
[70/359] Building C object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/fatal.c.obj
[71/359] Building C object spm/zephyr/arch/arm/core/CMakeFiles/spm_arch__arm__core.dir/nmi.c.obj
[72/359] Building C object spm/zephyr/arch/arm/core/cortex_m/CMakeFiles/spm_arch__arm__core__cortex_m.dir/scb.c.obj
[73/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/ext/hal/nordic/nrfx/drivers/src/nrfx_spim.c.obj
[74/359] Building C object spm/zephyr/arch/arm/core/cortex_m/CMakeFiles/spm_arch__arm__core__cortex_m.dir/thread_abort.c.obj
[75/359] Building C object spm/zephyr/arch/arm/core/cortex_m/CMakeFiles/spm_arch__arm__core__cortex_m.dir/irq_init.c.obj
[76/359] Building C object spm/zephyr/lib/libc/minimal/CMakeFiles/spm_lib__libc__minimal.dir/source/stdlib/malloc.c.obj
[77/359] Building C object spm/zephyr/arch/arm/core/cortex_m/mpu/CMakeFiles/spm_arch__arm__core__cortex_m__mpu.dir/arm_core_mpu.c.obj
[78/359] Building C object spm/zephyr/arch/arm/core/cortex_m/CMakeFiles/spm_arch__arm__core__cortex_m.dir/fault.c.obj
[79/359] Building C object spm/zephyr/CMakeFiles/spm_zephyr.dir/ext/hal/nordic/nrfx/drivers/src/nrfx_twim.c.obj
[80/359] Building C object spm/zephyr/arch/arm/core/cortex_m/mpu/CMakeFiles/spm_arch__arm__core__cortex_m__mpu.dir/arm_mpu.c.obj
ninja: build stopped: subcommand failed.
ERROR: command exited with status 1: /root/.local/bin/cmake --build /root/ncs/nrf/applications/cat_tracker/build
Exited with code 1
```

</details>

**Comment by coderbyheart** [comment]
Failing build: https://circleci.com/gh/bifravst/cat-tracker-fw/44?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by coderbyheart** [comment]
Fixed via e7041d3c0b9cfc0485926fa0a925e6d805ddb3d1

See https://github.com/bifravst/cat-tracker-fw/releases/tag/v0.8.1
